fedmsg.d/fmn.py

```diff
@@ -93,7 +93,12 @@
         'include': ['fmn.tasks'],
         'accept_content': ['json'],
         'task_serializer': 'json',
-        'task_default_queue': 'workers',
+        'task_routes': {
+            'fmn.tasks.find_recipients': {
+                'queue': 'fmn.tasks.unprocessed_messages',
+                'routing_key': 'fmn.tasks.unprocessed_messages',
```

Comment:
This change looks quite similar to the change made in ```ansible/roles/fmn-dev/files/fedmsg.d/fmn.py``` - is there an opportunity to share code here?

Reply:
Yes, but I'd prefer to put that off. My next PR will be refactoring the backend subpackage, followed by a configuration module with defaults and all that jazz. The next release is going to be fmn-2.0 and I'm going to take the opportunity to clean up the configuration.